doc/Command-Reference.md

```diff
@@ -8033,6 +8320,13 @@ This command is used to install a new image on the alternate image partition.  T
   Done
   ```
 
+SONiC image installation will install SONiC packages that are installed in currently running SONiC image. In order to perform clean SONiC installation use *--skip-package-migration* option when installing SONiC image:
+
+- Example:
+  ```
+  admin@sonic:~$ sudo sonic-installer install https://sonic-jenkins.westus.cloudapp.azure.com/job/xxxx/job/buildimage-xxxx-all/xxx/artifact/target/sonic-xxxx.bin --skip-package-migration
```

Comment:
Will it upgrade packages, if currently installed is lower version, w/o --skip flag?

Reply:
yes, please refer to HLD - https://github.com/stepanblyschak/SONiC/blob/sonic-app-ext-3/doc/sonic-application-extention/sonic-application-extention-hld.md#sonic-2-sonic-upgrade